src/pages/OperatorsPage.tsx

```diff
@@ -101,7 +103,9 @@ export const OperatorsPage = () => {
                         <Button
                             onClick={async () => {
                                 try {
-                                    await becomeOperatorModal.pop()
+                                    await becomeOperatorModal.pop({
+                                        onSubmit: createOperator,
```

Comment:
Should we just remove `onSubmit` and keep the logic inside `BecomeOperatorModal.tsx`?

Reply:
some time ago we were discussing with Mariusz that we'd like to keep the modals without any application logic